site/src/esl-popup/esl-d-popup-game.ts

```diff
@@ -42,14 +42,14 @@ export class ESLDemoPopupGame extends ESLBaseElement {
     event.preventDefault();
   }
 
-  @listen({auto: false, event: 'mouseleave mouseup'})
+  @listen({auto: false, event: 'mouseup', target: window})
   protected _onDragStop(): void {
     this.dragging = false;
     this.$$off(this._onDragging);
     this.$$off(this._onDragStop);
   }
 
-  @listen({auto: false, event: 'mousemove'})
+  @listen({auto: false, event: 'mousemove', target: window})
   @decorate(rafDecorator)
```

Comment:
```suggestion
```

Reply:
Maybe this video - [I never thought of using CSS animations like this before!](https://www.youtube.com/watch?v=IdxzJLQ3Mbs) can be usefull as source of ideas for animations?